src/luascript.h

```diff
@@ -240,7 +240,7 @@ class LuaScriptInterface
 		bool pushFunction(int32_t functionId);
 
 		static int luaErrorHandler(lua_State* L);
-		bool callFunction(int params);
+		bool callFunction(int params, bool defaultReturn = false);
```

Comment:
It's true, but only when called from the onStep functions, for the rest it should be false